<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCubeRTC</class>
 <widget class="QWidget" name="GameCubeRTC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameCube RTC</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="TextBox" name="textBoxStartSeed"/>
   </item>
   <item row="0" column="1">
    <widget class="TextBox" name="textBoxTargetSeed"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelStartSeed">
     <property name="text">
      <string>Seed at 1/1/2000 at 00:00:00</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelEndSeed">
     <property name="text">
      <string>Target Seed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="TextBox" name="textBoxMinAdvance">
     <property name="text">
      <string notr="true">500000</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="TextBox" name="textBoxMaxAdvance">
     <property name="text">
      <string notr="true">1000000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelMinAdvance">
     <property name="text">
      <string>Min Advance</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelMaxAdvance">
     <property name="text">
      <string>Max Advance</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButtonSearch">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="TableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textBoxStartSeed</tabstop>
  <tabstop>textBoxTargetSeed</tabstop>
  <tabstop>textBoxMinAdvance</tabstop>
  <tabstop>textBoxMaxAdvance</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
